controllers/statisticsController.ts

```diff
@@ -1,18 +1,30 @@
 import { Request, Response } from 'express';
-import { getUniversityCount, getActivePsychologistCount, getPatientCount } from '../db/statistics';
+import {
+  getUniversityCount, getAppointmentCount, getActivePsychologistCount, getPatientCount,
+} from '../db/statistics';
 import asyncHelper from '../utils/async-helper';
 
 const getAll = async (req: Request, res: Response): Promise<void> => {
   const university = getUniversityCount();
+  const appointments = getAppointmentCount();
   const psychologist = getActivePsychologistCount();
   const patients = getPatientCount();
 
-  const [universityCount, psychologistCount, patientsCount] = await Promise.all([university, psychologist, patients]);
+  const [
+    universityCount,
+    appointmentsCount,
+    psychologistCount,
+    patientsCount,
+  ] = await Promise.all([university, appointments, psychologist, patients]);
   res.json([
     {
       label: 'Universités partenaires',
       value: universityCount[0].count,
     },
+    {
+      label: 'Nombre de séances réalisées',
```

Comment:
```suggestion
      label: 'Séances réalisées',
```
to use same wording as the other stats